chart/templates/workers/worker-deployment.yaml

```diff
@@ -232,7 +233,13 @@ spec:
           {{- if .Values.workers.command }}
           command: {{ tpl (toYaml .Values.workers.command) . | nindent 12 }}
           {{- end }}
-          {{- if .Values.workers.args }}
+          {{- if and .Values.workers.kerberosSidecar.enabled (semverCompare "<1.28" $kubeVersion) }}
+          args:
+            {{ tpl (toYaml .Values.workers.args) . | nindent 12 }} &&
```

Comment:
I think this will not work as expected for two reasons:

1) what you can see in the test - `exec airflow celery worker` will replace the bash interpreter with the python one (this is how exec works) - this means that whatever is after `&&` will not be executed because bash interpreter will not be there any more to execute it 

2) Even if it did work and we had no exect, this would not work if the worker crashed and exited with error - because in this case what's after `&&` is not executed.

Reply:
Thanks for that @potiuk
Looking for alternatives, any ideas on this front?

Reply:
I tried something like this:
```
          {{- if and .Values.workers.kerberosSidecar.enabled (semverCompare "<1.28" $kubeVersion) }}
          args:
            - {{ tpl (toYaml .Values.workers.args) . | nindent 12 }}
            - "&&"
            - "sh"
            - "-c"
            - >
              echo "EXIT file content" > /opt/kerberos-exit/EXIT
          {{- else if .Values.workers.args }}
          args: {{ tpl (toYaml .Values.workers.args) . | nindent 12 }}
          {{- end }}
```

This evaluates to:
```
          args:
            - 
            - bash
            - -c
            - |-
              exec \
              airflow celery worker
            - "&&"
            - "sh"
            - "-c"
            - >
              echo "EXIT file content" > /opt/kerberos-exit/EXIT
```

Reply:
And adding a trim in helm makes it like this:
```
          args:
            - - bash
            - -c
            - |-
              exec \
              airflow celery worker
            - "&&"
            - "sh"
            - "-c"
            - >
              echo "EXIT file content" > /opt/kerberos-exit/EXIT
```

Thinking of some alternatives :)